Herebyfile.mjs

```diff
@@ -854,3 +855,14 @@ export const help = task({
     hiddenFromTaskList: true,
     run: () => exec("hereby", ["--tasks"], { hidePrompt: true }),
 });
+
+export const bumpLkgToNightly = task({
+    name: "bump-lkg-to-nightly",
+    description: "Bumps typescript in package.json to the latest nightly and copies it to LKG.",
+    run: async () => {
+        await exec("npm", ["install", "--save-dev", "--save-exact", "typescript@next"]);

```

Comment:
This hardcodes `@next`, but feasibly we may want (after 5.0 release) to switch to a stable build. But, I suspect that by then, we'll have deleted LKG anyway.

Reply:
Presumably we keep manually bumping the LKG in `main` for now?

Reply:
Yes, when required for one reason or another.

Reply:
But, this won't be updated in the nightly package-lock update, since they'd get out of sync, and keeping them in sync so will create loads of churn (the antithesis of what we're hoping to get out of this ordeal).